src/vs/base/common/encoding.ts

```diff
@@ -3,6 +3,8 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+// FIX BEFORE MERGE
+// eslint-disable-next-line code-import-patterns
```

Comment:
In the `.eslintrc.json` there is a comment asking to avoid relaxing constraints. That's why I disabled line here to be able to commit and would like to ask for your guidance.

From my perspective we can add `iconv-lite-umd`, it seems to be an exceptional case.

Reply:
@gyzerok this is fixable by allowing `iconv-lite-umd` for this particular case. Simply check for how it's done for `semver-umd`:

https://github.com/microsoft/vscode/blob/8cd3fe080bee38d28ba01d0d71713d6ec22b254e/.eslintrc.json#L629